keyRegistration/js/registration.js

```diff
@@ -66,7 +65,7 @@ $(document).ready(function()
        },
     });
 
-    function setError(text){
+    function setError(text) {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*